taskApp/src/app/app-routing.module.ts

```diff
@@ -3,11 +3,14 @@ import { Routes, RouterModule } from '@angular/router';
 import { BrowserModule } from '@angular/platform-browser';
 import { TaskViewComponent } from './pages/task-view/task-view.component';
 import { NewCatComponent } from './pages/new-cat/new-cat.component';
+import { ListTasksComponent } from './pages/list-tasks/list-tasks.component';
+
 
 const routes: Routes = [
   { path: '', redirectTo: '/cats', pathMatch:'full'},
   { path: 'new-cat', component: NewCatComponent},
   { path: 'cats', component: TaskViewComponent },
```

Comment:
Can I ask why these are named cats???

Reply:
'cats' is supposed to be for the categories. I never implemented full functionality on them, I've been working on just the tasks

Reply:
Oh wow for some reason my brain wasn't equating cats to categories lol.  Ok, thank you for the clarification!